Model/Sales/Repository/PackageRepository.php

```diff
@@ -252,6 +258,25 @@ public function getProductDropOffDelay(array $products): ?int
         return $highestDropOffDelay > 0 ? $highestDropOffDelay : null;
     }
 
+    /**
+     * @param array  $products
+     * @param string $carrierPath
+     *
+     * @return bool
+     */
+    public function getAgeCheck(array $products, string $carrierPath): bool
+    {
+        foreach ($products as $product) {
+            $productAgeCheck  = (bool) $this->getAttributesProductsOptions($product, 'age_check');
+
+            if ($productAgeCheck) {
+                return true;
+            }
+        }
+
+        return (bool) $this->getConfigValue($carrierPath . 'default_options/age_check_active');
```

Comment:
Nitpicking, maar als dit waar is, hoef je niet door die products te gaan toch? Zou je die dan niet als eerste willen checken?

Reply:
in deze method worden de gegevens opgehaald van de producten en van de default installing. Dit is dus nodig om de default instelling op te halen.

Reply:
Daarbij is de volgorde dat er eerst gekeken moet worden naar de producten. Als daar true staat dan maakt het niet uit wat je bij de default instellingen heb staan.

Reply:
Je hebt gelijk natuurlijk!